Client/core/CChat.cpp

```diff
@@ -57,6 +57,11 @@ CChat::CChat ( CGUI* pManager, const CVector2D & vecPosition )
     m_pCacheTexture = NULL;
     m_iCacheTextureRevision = 0;
     m_iReportCount = 0;
+    m_fPositionOffsetX = 0.0125;
+    m_fPositionOffsetY = 0.0150;

```

Comment:
These 2 lines cause a warning: `C4305: '=': truncation from 'double' to 'float'`. Append an `f` as a suffix.